domino_test.go

```diff
@@ -137,11 +139,11 @@ func TestGetItemEmpty(t *testing.T) {
 	err := table.CreateTable().ExecuteWith(ctx, db)
 	defer table.DeleteTable().ExecuteWith(ctx, db)
 
-	assert.Nil(t, err)
+	assert.NoError(t, err)
 
-	user, err := table.GetItem(KeyValue{"naveen@email.com", "password"}).ExecuteWith(ctx, db, &User{})
-	assert.Nil(t, err)
-	assert.Nil(t, user)
+	out := table.GetItem(KeyValue{"naveen@email.com", "password"}).ExecuteWith(ctx, db)
```

Comment:
The 'out' object is a domino output composed with the dynamodb GetItemOutput object. You still need to call Result to deserialize to a domain object, but you can interact with the GetItemOutput fi odd/methods directly aswell.